src/UrqlTypes.re

```diff
@@ -82,4 +82,21 @@ type operationResult = {
   data: operationData,
   [@bs.optional]
   error: UrqlCombinedError.t,
-};
\ No newline at end of file
+};
+
+/* The signature of the structure created by calling `.make()` on a `graphql_ppx` module. */
+type request('response) = {
```

Comment:
We'll want to use this type in `useQuery` and `useMutation` cc/ @gugahoa @Schmavery.